src/app/features/vendor/models/vendor.model.ts

```diff
@@ -3,11 +3,11 @@ export interface IVendor {
   name: string;
   state?: string;
   country: string;
+  code: string;
```

Comment:
is this required?

Reply:
Its in the dev branch IVendor interface, but now not needed since phonenumber prefix code dropdown is removed.

Reply:
Removed